File: leetcode/3005. 最大频率元素计数.py
class Solution:
    def maxFrequencyElements(self, nums: List[int]) -> int:
        m = dict()
        for n in nums:
            m[n] = m.get(n, 0) + 1
        maxCount = 0
        res = 0
        for k, v in m.items():
            if v > maxCount:
                maxCount = v
        for k, v in m.items():
            if v == maxCount:
                res += maxCount

        return res

File: leetcode/3005. 最大频率元素计数.go
func maxFrequencyElements(nums []int) int {
	m := make(map[int]int)
	maxCount := 0
	res := 0
	for _, num := range nums {
		m[num] += 1
	}
	for _, count := range m {
		if count > maxCount {
			maxCount = count
		}
	}
	for _, count := range m {
		if count == maxCount {
			res += maxCount
		}
	}
	return res
}